saagieapi/pipelines/gql_queries.py

```diff
@@ -134,141 +152,167 @@
         scheduleStatus
         scheduleTimezone
         isLegacyPipeline
+        hasExecutionVariablesEnabled
     }
 }
-  """
+"""
 
 GQL_STOP_PIPELINE_INSTANCE = """
 mutation stopPipelineInstanceMutation($pipelineInstanceId: UUID!){
     stopPipelineInstance(pipelineInstanceId: $pipelineInstanceId){
-      id
-      number
-      status
-      startTime
-      endTime
-      pipelineId
+        id
+        number
+        status
+        startTime
+        endTime
+        pipelineId
     }
 }
-  """
+"""
 
 GQL_EDIT_PIPELINE = """
-  mutation($id: UUID!, $name: String, $description: String, $alerting: JobPipelineAlertingInput,
-          $isScheduled: Boolean, $cronScheduling: Cron, $scheduleTimezone:TimeZone)  {
-    editPipeline(pipeline: {
-        id: $id
-        name: $name
-        description: $description
-        alerting:  $alerting
-        isScheduled: $isScheduled
-        cronScheduling: $cronScheduling
-        scheduleTimezone: $scheduleTimezone
-      })
-    {
-      id
-      name
-      description
-      alerting{
-        emails
-        statusList
-      }
-      isScheduled
-      cronScheduling
-      scheduleTimezone
+mutation($id: UUID!, 
+		 $name: String, 
+		 $description: String, 
+		 $alerting: JobPipelineAlertingInput,
+		 $isScheduled: Boolean, 
+		 $cronScheduling: Cron, 
+		 $scheduleTimezone:TimeZone, 
+		 $hasExecutionVariablesEnabled: Boolean)    {
+    editPipeline(pipeline: 
+        {
+            id: $id
+            name: $name
+            description: $description
+            alerting:    $alerting
+            isScheduled: $isScheduled
+            cronScheduling: $cronScheduling
+            scheduleTimezone: $scheduleTimezone
+            hasExecutionVariablesEnabled: $hasExecutionVariablesEnabled
+        }
+    ){
+        id
+        name
+        description
+        alerting{
+            emails
+            statusList
+        }
+        isScheduled
+        cronScheduling
+        scheduleTimezone
+        hasExecutionVariablesEnabled
```

Comment:
The doc of the section `Editing a pipeline` should be updated with the field `hasExecutionVariablesEnabled`